docs/configuration/advanced/misc.md

```diff
@@ -25,17 +25,6 @@ Default: 16384
     read_buffer_size: 16384
 
 
-## New-Terminal spawn behaviour
-
-This flag determines whether to spawn new process or not when creating new terminal
-
-If this option is set to `false`, then simply a new terminal window is being
-created rather thena fully creating a new process.
-
-Default: `false`
-
-    spawn_new_process: false
-
```

Comment:
Note that, as discussed on Discord, this will be a step back in terms of supporting proper expected behaviour in the case of MacOS.

Reply:
Yes, will cancel pr